lib/rbs/definition_builder/ancestor_builder.rb

```diff
@@ -321,6 +321,7 @@ def mixin_ancestors0(decl, align_params:, included_modules:, included_interfaces
 
             case
             when member.name.class? && included_modules
+              InvalidMixinClassError.check!(mixin_name: "include", env: env, member: member)
```

Comment:
The `mixin_name` argument seems redundant.
It would be better to detect the _type_ of the mixin from `member`.

```
case member
when AST::Members::Prepend
  :prepend
when AST::Members::Include
  :include
when AST::Members::Extend
  :extend
end
```